src/fulltrade/FullTradeParams.js

```diff
@@ -28,10 +34,13 @@ import { createDefaultType, createDefaultDuration, createDefaultBarriers } from
  * 3. ticks is always within 5 to 10
  * 4. digit trade is always ticks only
  * 5. barriers are non available for contract below 2 minutes
- * 6. forward starting does not have barriers
+ * 6. For underlying that's under our control, we allow relative and absolute barrier regardless of intraday or not
+ * 7. Relative barrier is always allowed
+ * 8. Forward starting do not have barriers
+ * 9. All underlying have risefall trade
  *
  *
- * TODO: reform the whole dependency mess,
+ * TODO: re-architect the whole dependency mess,
```

Comment:
most of this assumption is assumption on data, meaning we are short of metadata, we need to confirm with Quants, and think what to do when assumption breaks